glamour.go

```diff
@@ -207,6 +220,15 @@ func (tr *TermRenderer) RenderBytes(in []byte) ([]byte, error) {
 	return buf.Bytes(), err
 }
 
+func getGlamourStyleFromEnvironment() string {
```

Comment:
Maybe we should simply call this `getStyleFromEnvironment` or `getEnvironmentStyle` to be more in line with the `getDefaultStyle` function below?

Reply:
Ah. Yeah. That does seem cleaner. Renamed it to `getEnvironmentStyle`.